### PR TITLE
docs(angular): some improvements to the Nx vs Angular CLI page

### PR DESCRIPTION
**Pull request by juristr**
Improves the page (https://nx-dev-git-fork-juristr-docs-angular-cli-nx-improvement-nrwl.vercel.app/concepts/more-concepts/nx-and-angular)
- by having 3 main headings emphasizing the why & migration scenarios
- nesting the details about similarities & differences into a subsection
- slighly adjusting the comparison table

### CODE REVIEW

File: docs/shared/guides/nx-and-angular-cli.md [hunk 1]
```diff
@@ -322,24 +340,28 @@ As you start modularizing your Angular workspace, Nx can visualize it using the
...
+
+```shell
+npx nx@latest init
+```
 
-## Being the best CLI for you!
+or alternatively using the `--integrated` flag if you want to create an Nx monorepo right away. Learn more about all the details on the [dedicated docs page](/recipes/angular/migration/angular).
 
-The Nx team's only mission is to build the best CLI for you. We don't build frameworks or libraries; we make tools for them!
+There's also a guide how to
```
Comment (isaacplmann) [inline docs/shared/guides/nx-and-angular-cli.md:359]:
```suggestion
There are also guides describing how to:
```

File: docs/shared/guides/nx-and-angular-cli.md [hunk 2]
```diff
@@ -322,24 +340,28 @@ As you start modularizing your Angular workspace, Nx can visualize it using the
...
 
-## Convinced?
+- [manually migrate an Angular CLI app](/recipes/angular/migration/angular-manual) and
+- how to [consolidate multiple Angular CLI projects into a single Nx monorepo](/recipes/angular/migration/angular-multiple)
```
Comment (isaacplmann) [inline docs/shared/guides/nx-and-angular-cli.md:362]:
```suggestion
- [consolidate multiple Angular CLI projects into a single Nx monorepo](/recipes/angular/migration/angular-multiple)
```

File: docs/shared/guides/nx-and-angular-cli.md [hunk 3]
```diff
@@ -322,24 +340,28 @@ As you start modularizing your Angular workspace, Nx can visualize it using the
...
 
-If you're convinced, then here are some options for you:
+You can also learn more about Angular & Nx following our dedicated tutorials:
```
Comment (isaacplmann) [inline docs/shared/guides/nx-and-angular-cli.md:364]:
```suggestion
You can learn more about Angular & Nx by following our dedicated tutorials:
```

**Review by isaacplmann — CHANGES_REQUESTED** [review]
Small changes

File: docs/shared/guides/nx-and-angular-cli.md [hunk 4]
```diff
@@ -7,36 +7,54 @@ width="100%" /%}
...
+| Support for [single-project Workspaces](/getting-started/tutorials/angular-standalone-tutorial)            | ✅              | ✅            |
+| First-Class [Monorepo Support](/getting-started/tutorials/angular-monorepo-tutorial)                       | ❌\*            | ✅            |
+| [Enforced Module Boundaries](/core-features/enforce-module-boundaries)                                     | ❌              | ✅            |
+| Interactive [Project Graph](/core-features/explore-graph)                                                  | ❌              | ✅            |
+| Task Graph                                                                                                 | ❌              | ✅            |
+| [Running Tasks in Parallel](/recipes/running-tasks/run-tasks-in-parallel)                                  | ❌              | ✅            |
+| Building, Testing [Only What is Affected](/core-features/run-tasks#run-tasks-on-projects-affected-by-a-pr) | ❌              | ✅            |
+| [Local Caching](/core-features/cache-task-results)                                                         | ❌              | ✅            |
```
Comment (bulldog98) [inline docs/shared/guides/nx-and-angular-cli.md:45]: To be fair angular has local caching, since a few versions, see: https://angular.io/cli/cache

**Review by isaacplmann — APPROVED** [review]

File: docs/shared/guides/nx-and-angular-cli.md (hunk 4 above)
Comment (juristr) [reply docs/shared/guides/nx-and-angular-cli.md:45]: yeah but this is a little different. This is - unless it changed - the webpack level persistent cache that gets written into the `.ng` location to speed up rebuilds. The Nx cache is agnostic and actually leverages that cache underneath as well.

Comment (juristr) [reply docs/shared/guides/nx-and-angular-cli.md]: @bulldog98 I added some clarification. Thanks

**Pull request merged**